includes/abstracts/abstract-batch.php

```diff
@@ -88,6 +88,13 @@
 	public $total_num_results;
 
 	/**
+	 * Difference in number of results between steps
+	 *
+	 * @var int
+	 */
+	public $results_changed = 0;
```

Comment:
What do you think of `$total_results_difference` or something? I noticed as I was reading below the var name `$results_changed` seemed confusing without more context (specifically within `calculate_offset`).